tests/onnxruntime/test_modeling.py

```diff
@@ -61,6 +61,31 @@ def test_load_model_from_hub(self):
         model = ORTModel.from_pretrained(self.ONNX_MODEL_ID)
         self.assertIsInstance(model.model, onnxruntime.capi.onnxruntime_inference_collection.InferenceSession)
         self.assertIsInstance(model.config, PretrainedConfig)
+        self.assertListEqual(model.providers, ["CPUExecutionProvider"])
+        self.assertEqual(model.device, torch.device("cpu"))
+
+    def test_load_model_cuda_provider_list(self):
```

Comment:
are those tests run on any device or noly when there is a GPU available?

Reply:
Good catch, I use the decorator `@require_torch_gpu` now.